presto-main/src/main/java/io/prestosql/server/PrestoServer.java

```diff
@@ -114,6 +115,9 @@ public void run()
         try {
             Injector injector = app.strictConfig().initialize();
 
+            log.info("Working directory: %s", Paths.get(".").toAbsolutePath().toRealPath());
+            log.info("Etc directory: %s", Paths.get("etc").toAbsolutePath().toRealPath());
```

Comment:
Maybe we should also log `com.facebook.presto.server.PluginManagerConfig#getInstalledPluginsDir`?
And location of `var` directory, but I don't know how to get it.

Reply:
Also:
 * what is the point of loggin`.`, it is "already logged" with logging of `etc`.
 * Maybe it should be logged before `app.strictConfig().initialize();`, so it case of its failure we might have more info to diagnose why it failed.

Reply:
> Maybe it should be logged before app.strictConfig().initialize();, so it case of its failure we might have more info to diagnose why it failed.
 
i will move

> what is the point of loggin., it is "already logged" with logging of etc.

`.` can be eg `/var/lib/presto/...`
`etc` can be eg `/etc/presto`